collectors/gsuite/README.md

```diff
@@ -10,35 +10,57 @@ forward logs to the Alert Logic CloudInsight backend services.
 
 # Installation
 
-Refer to [CF template readme](./cfn/README.md) for installation instructions.
+### 1. G Suite Setup
+
+1. Create a Custom admin role. [Link](https://support.google.com/a/answer/1219251?hl=en&ref_topic=4514341)
+2. This admin role should have ‘Reports’ privilege.
```

Comment:
Could you merge this point and the one above? Something like `Create a custom admin role with the 'Reports' privilege assigned`